src/test/java/org/wso2/carbon/connector/integration/test/cerner/CernerConnectorIntegrationTest.java

```diff
@@ -0,0 +1,396 @@
+     */
+    @Test(enabled = true, description = "cerner {searchProcedure} integration test with mandatory parameters.")
+    public void searchProcedurewithMandatoryParameters() throws IOException, JSONException {
+        eiRequestHeadersMap.put("Action", "urn:searchProcedure");
+        RestResponse<JSONObject> eiRestResponse = sendJsonRestRequest(proxyUrl, "POST", eiRequestHeadersMap, "searchProcedure.json");
+        final String apiEndPoint = connectorProperties.getProperty("base") + "/" + connectorProperties.getProperty("procedureType") +
+                "?patient=" + connectorProperties.getProperty("patient");
+        RestResponse<JSONObject> apiRestResponse = sendJsonRestRequest(apiEndPoint, "GET", apiRequestHeadersMap);
+        Assert.assertEquals(eiRestResponse.getHttpStatusCode(), 200,"Invalid parameter value");
+        Assert.assertEquals(apiRestResponse.getHttpStatusCode(), 200,"Invalid parameter value");
+    }
+
+    /**
+     * Positive test case for searchRelatedPerson method with Mandatory parameters.
+     */
+    @Test(enabled = true, description = "cerner {searchRelatedPerson} integration test with mandatory parameters.")
+    public void searchRelatedPersonwithMandatoryParameters() throws IOException, JSONException {
+        eiRequestHeadersMap.put("Action", "urn:searchRelatedPerson");
+        RestResponse<JSONObject> eiRestResponse = sendJsonRestRequest(proxyUrl, "POST", eiRequestHeadersMap, "searchRelatedPerson.json");
+        final String apiEndPoint = connectorProperties.getProperty("base") + "/" + connectorProperties.getProperty("relatedPersonType") +
+                "?patient=" + connectorProperties.getProperty("patient");
+        RestResponse<JSONObject> apiRestResponse = sendJsonRestRequest(apiEndPoint, "GET", apiRequestHeadersMap);
+        Assert.assertEquals(eiRestResponse.getHttpStatusCode(), 200,"Invalid parameter value");
+        Assert.assertEquals(apiRestResponse.getHttpStatusCode(), 200,"Invalid parameter value");
+    }
+
+    /**
+     * Positive test case for searchSlot method with Mandatory parameters.
+     */
+    @Test(enabled = true, description = "cerner {searchSlot} integration test with mandatory parameters.")
+    public void searchSlotwithMandatoryParameters() throws IOException, JSONException {
+        eiRequestHeadersMap.put("Action", "urn:searchSlot");
+        RestResponse<JSONObject> eiRestResponse = sendJsonRestRequest(proxyUrl, "POST", eiRequestHeadersMap, "searchSlot.json");
+        final String apiEndPoint = connectorProperties.getProperty("base") + "/" + connectorProperties.getProperty("slotType") +
+                "?_id=" + connectorProperties.getProperty("slotId");
+        RestResponse<JSONObject> apiRestResponse = sendJsonRestRequest(apiEndPoint, "GET", apiRequestHeadersMap);
+        Assert.assertEquals(eiRestResponse.getHttpStatusCode(), 200,"Invalid parameter value");
+        Assert.assertEquals(apiRestResponse.getHttpStatusCode(), 200,"Invalid parameter value");
```

Comment:
In all the tests, we are checking the status code only. It's good if we compare some response fields.